feat(content): The Successors - Enter the Frogtopus Aristocracy (#10296)
endless-sky/endless-sky: data/successors/successor news.txt

# Copyright (c) 2024 by Daeridanii
#
# Endless Sky is free software: you can redistribute it and/or modify it under the
# terms of the GNU General Public License as published by the Free Software
# Foundation, either version 3 of the License, or (at your option) any later version.
#
# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A

> # PARTICULAR PURPOSE.  See the GNU General Public License for more details.

Reviewer: ```suggestion
# PARTICULAR PURPOSE. See the GNU General Public License for more details.
```

#
# You should have received a copy of the GNU General Public License along with
# this program. If not, see <https://www.gnu.org/licenses/>.

> 

Reviewer 2: ```suggestion

```

news "high houses scion"
	to show
		has "language: Successor"
	location
		attributes successor
		attributes wealthy
		not attributes uninhabited
	name
		word
			"High Houses Scion"
			"Child of the High Houses"
			"Scion of the High Houses"

> 	message

Reviewer 2: Normally what happens is that there is a quotation mark at the beginning and end in separate word blocks, so you may want to do that here.

		word
			`"I hope thou has't enjoyed thy time amongst our people."`
			`"The alien! Doth thou have news from far in this endless sky?"`
			`"Fair traveler, I've heard of thee. How must thy voyage weigh upon thy mind."`
			`"Welcome to our space, honored alien. A pleasure 'tis to meet thee."`

> 

Reviewer 2: ```suggestion

```

news "high houses official"
	to show
		has "language: Successor"
	location
		attributes successor
		not attributes crime uninhabited
	name
		word
			"High Houses official"
			"High Houses officer"
	message
		word
			`"The Houses welcome thee as 'guest' among our people; `
			`"Please remain in concord with our people; `
			`"Informed, I am of thy recent arrival; `
			`"Can I provide assistance, honored traveler? `
			`"Our noble house does greet thee full of joy; `
		word
			`I am at thy service if 'tis needed."`
			`Today's been rather busy, please speak quickly."`
	message
		word
			`"Please submit to scans if 'tis requested`
			`"Taxes on transactions are collected`
			`"Transactions in this area are surveilled`
			`"If thou see'st something suspect, report't promptly`
		word
			`; port rules and regulations should be posted."`
			` in accordance with our standard port procedure."`

> 

Reviewer 2: ```suggestion

```

news "successor student"
	to show
		has "language: Successor"
	location
		attributes successor
		attributes education wealthy
		not attributes uninhabited
	name
		word
			"Successor student"
	message
		word
			`"Exams, I think, shall never know their end, `
			`"Hath I forgot my geographic terms? `
			`"Hath I forgot my chemical reagents? `
			`"Hyperspace Mechanics is a bore, `
			`"Predecessors His'try is a drag, `
			`"Plasma Flow Dynamics is a bore, `
			`"Particle Kinetics is a bore, `
		word
			`I know not sleep nor food nor meager rest`
			`I wish that I could learn while I'm asleep`
			`I don't know how I even stay awake`
			`I wish I knew some better study tricks`
			`but I know that I'm the best one in my class`
		word
			`."`
			`!"`

> 

Reviewer 2: ```suggestion

```

	name
		word
			"Independent merchant"
	message
		word
			`"Interested in `
		phrase
			"2-syllable commodities"
		word
			`, noble traveler?"`
			`, canny buyer?"`
			`, noble captain?"`
			`, clever buyer?"`
			`, newest friend?"`
			`, thoughtful buyer?"`
			`, thoughtful captain?"`
	message
		word
			"I miss the days when I had been a "
			"I miss the times when I had been a "
			"I still think back to when I was a "
			"Thou remindeth me of when I was a "
		word
			"pilot, "
			"captain, "
			"ranger, "
		word
			"but alas "
		word
			"now's planetside I shall remain."
			"'twas never brave enough to stay."
			"I was a spacesick little knave."
			"it's down at earth I feel at home."
			"it's selling items that's for me."
			"my family needed me return."
			"possessed of talent I was not."
	message
		word
			"A "
		word
			"pushy "
			"unscrupulous "
			"friendly-looking "
			"anxious-looking "
			"furtive "
			"brightly-colored "
			"vibrantly-dressed "
			"dour-looking "
		word
			"merchant "
			"trader "
			"artisan "
			"Successor "
		word
			"tries to get you to "
		word
			"purchase "
			"buy "
		word
			"some "
		phrase
			"2-syllable commodities"
		word
			"."

> 

Reviewer 2: ```suggestion

```

news "high houses propaganda"
	to show
		has "language: Successor"
	location
		attributes successor
		not attributes quiet uninhabited
	name
		word
			"High Houses broadcast"
	message
		word
			`An advertisement shows a Successor warship performing a variety of deftly-executed maneuvers before encouraging the viewer to enlist in the armed forces of one of the High Houses.`
			`An advertisement plays showing a group of Successors standing close together against a factory backdrop, their skins all tinted a happy green. It is followed by a voiceover noting that wages have been recently increased.`
			`A short video plays showing the construction of a new spaceport on the surface of a prospering Successor planet. Several warships of the High Houses painted in ceremonial livery pass across the screen, showing clearly who is to be thanked.`

> 			`A proud-looking Successor naval officer stands before an imposing-looking warship: "Enlist!" the caption reads, "See the stars today!"`

Reviewer 2: ```suggestion
			`A proud-looking Successor naval officer stands before an imposing-looking warship: "Enlist!" the caption reads. "See the stars today!"`
```

			`An field of large, floral plants are shown growing long strings of gleaming metal which are quickly harvested by agricultural robots.`
			`A series of images displaying several models flash one after another so quickly that it is hard to tell what is actually being advertised.`
			`A short video clip is shown advertising a performance in a local concert hall. The stage is divided into two sections: one is underwater and the other is dry.`
			`Some sort of interactive game that you can't make sense of captivates the attention of a group of young Successors.`
			`An eye covering vaguely resembling a contact lens is shown being applied to the eyes of several models, changing their color and shape when applied.`
			`A shipbuilding organization advertises a members' discount on Kvelqde shuttles. They make sure to note that their operation is supported by one of the High Houses.`
			`An independent pilot briefly advertises their courier services in a pre-recorded message, offering guaranteed fastest-possible-delivery on packages under ten tons.`

> 

Reviewer 2: ```suggestion

```

# Copyright (c) 2024 by Daeridanii
#
# Endless Sky is free software: you can redistribute it and/or modify it under the
# terms of the GNU General Public License as published by the Free Software
# Foundation, either version 3 of the License, or (at your option) any later version.
#
# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
# PARTICULAR PURPOSE. See the GNU General Public License for more details.
#
# You should have received a copy of the GNU General Public License along with
# this program. If not, see <https://www.gnu.org/licenses/>.

news "high houses scion"
	to show
		has "language: Successor"
	location
		attributes successor
		attributes wealthy
		not attributes uninhabited
	name
		word

> 			"High Houses Scion"

Author: Hmm, I'm not certain this news item should exist at all. As I've described them, the scions of the various Houses tend to be aloof enough that it's unlikely FL would be running into them in the spaceport.

		word
			`"`
		word
			"The Houses welcome thee as 'guest' among our people; "
			"Please remain in concord with our people; "
			"Informed, I am, of thy recent arrival; "
			"Can I provide assistance, honored traveler? "
			"Our noble house does greet thee full of joy; "
		word
			"I am at thy service if 'tis needed."
			"Today's been rather busy, please speak quickly."
		word
			`"`
	message
		word
			`"`
		word
			"Please submit to scans if 'tis requested"
			"Taxes on transactions are collected"
			"Transactions in this area are surveilled"
			"If thou see'st something suspect, report't promptly"
		word
			"; port rules and regulations should be posted."
			" in accordance with our standard port procedure."
		word
			`"`

news "successor student"
	to show
		has "language: Successor"
	location
		attributes successor
		attributes education wealthy
		not attributes uninhabited
	name
		word
			"Successor student"
	message
		word
			`"`
		word
			"Exams, I think, shall never know their end,"
			"Hath I forgot my geographic terms?"
			"Hath I forgot my chemical reagents?"
			"Hyperspace Mechanics is a bore,"
			"Predecessors His'try is a drag,"
			"Plasma Flow Dynamics is a bore,"

> 			"Particle Kinetics is a bore,"

Reviewer 3: The first thing that came to me reading these was, it would be neat if the adjective used to describe the class was a pun related to the class itself (like Plasma Flow Dynamics being torpid or something like that, although it's also good as is.

		word
			" "
		word
			"I know not sleep nor food nor meager rest"
			"I wish that I could learn while I'm asleep"
			"I don't know how I even stay awake"
			"I wish I knew some better study tricks"

> 			"but I know that I'm the best one in my class"

Reviewer 3: Technically this last one works with the questions, but it's a bit disjointed. Maybe either remove it or shift the questions into their own message (which could then be fleshed out).

Author: Hmm yes, I remember it causing some goofy news messages in the past.

		word
			"."
			"!"
		word
			`"`

news "successor merchant"
	to show
		has "language: Successor"
	location
		attributes successor
		not attributes uninhabited
	name
		word
			"Independent merchant"
	message
		word
			`"`
		word
			"Interested in "
		phrase
			"successor commodities (2-syllable)"
		word
			", noble traveler?"
			", canny buyer?"
			", noble captain?"
			", clever buyer?"
			", newest friend?"
			", thoughtful buyer?"
			", thoughtful captain?"
		word
			`"`
	message
		word
			`"`
		word
			"I miss the days when I had been a "

> 			"I miss the times when I had been a "

Reviewer 3: Technically, they still have been a pilot, captain, or ranger, so this is a little confusing.

			"I still think back to when I was a "
			"Thou remindeth me of when I was a "
		word
			"pilot, "
			"captain, "
			"ranger, "
		word
			"but alas "
		word
			"now's planetside I shall remain."
			"'twas never brave enough to stay."
			"I was a spacesick little knave."
			"it's down at earth I feel at home."
			"it's selling items that's for me."
			"my family needed me return."
			"possessed of talent I was not."
		word
			`"`
	message
		word
			"A "
		word
			"pushy "

> 			"unscrupulous "

Reviewer 3: This one and `anxious-looking` don't work with the article `a`. I'd probably just remove them.

			"friendly-looking "
			"anxious-looking "
			"furtive "
			"brightly-colored "
			"vibrantly-dressed "
			"dour-looking "
		word
			"merchant "
			"trader "
			"artisan "
			"Successor "
		word
			"tries to get you to "
		word
			"purchase "
			"buy "
		word
			"some "
		phrase
			"successor commodities (2-syllable)"
		word

> 			"."

Reviewer 3: Usually spaceport news is either a description of something you're seeing, or actual dialogue, not a description of what you (or those around you) are doing. I'm not sure this message really works in that context.

Reviewer 4: I think it's fine.

news "high houses propaganda"
	to show
		has "language: Successor"
	location
		attributes successor
		not attributes quiet uninhabited
	name
		word

> 			"High Houses broadcast"

Reviewer 3: You could probably add a few more here
```suggestion
			"High House broadcast"
			"Propaganda broadcast"
			"Spaceport video screen"
```

			"A short video plays showing the construction of a new spaceport on the surface of a prospering Successor planet. Several warships of the High Houses painted in ceremonial livery pass across the screen, showing clearly who is to be thanked."
			`A proud-looking Successor naval officer stands before an imposing-looking warship: "Enlist!" the caption reads. "See the stars today!"`
			`Several short vignettes play: a group of Successors unearthing ruins, the inside of a laboratory, a calm pastoral scene. A voiceover speaks: "Uncovering our heritage; bettering our lives." Meticulously-dressed officials of the High Houses are present in each situation, presented as supportive overseers.`
			"A advertisement plays showing the sleek lines of a Successor warship, its polished armor gleaming in the sunlight. It goes on for several minutes, showing the vessel from numerous angles while a sweeping soundtrack plays."

news "successor advertisements"
	to show
		has "language: Successor"
	location
		attributes successor
		not attributes quiet uninhabited
	name
		word

> 			"Advertisement"

Reviewer 3: Same as above

			"A cluster of small recreational sailing ships are shown on a pristine lake at sunset, their Successor crew either manning them or splashing happily in the water. A caption at the bottom encourages the viewer to purchase their own."
			"An advertisement plays for the Successor Pulse Laser: it is shown being constructed in a clean, bright factory, installed on a ship, and extracting minerals from asteroids. It is never shown in a combat situation."
			"An advertisement plays showing a variety of attractive and expensive-looking jewellery, made of gold, silver, and other stranger metals which you cannot recognize. A variety of models display it wrapped around their arms or other parts of their body, gently undulating as the camera moves."
			"A display shows an advertisement for one of the Successor universities, showing elegant, sculptural buildings emerging from the ground under a rain-soaked sky."
			"An field of large, floral plants are shown growing long strings of gleaming metal which are quickly harvested by agricultural robots."

> 			"A series of images displaying several models flash one after another so quickly that it is hard to tell what is actually being advertised."

Reviewer 3: Several models of what? Because this sounds like they're selling sculptures.

Author: Ah, "model" as in a profession/a person who 'models' a product. Could use some disambiguation.

			"A short video clip is shown advertising a performance in a local concert hall. The stage is divided into two sections: one is underwater and the other is dry."

> 			"Some sort of interactive game that you can't make sense of captivates the attention of a group of young Successors."

Reviewer 3: Is this still an advertisement? If so, it might make sense to make it more clear, and maybe focus on the actions of the Successors rather than their emotions (or even just say `the Successors appear captivated`).

			"An eye covering vaguely resembling a contact lens is shown being applied to the eyes of several models, changing their color and shape when applied."

> 			"A shipbuilding organization advertises a members' discount on Kvelqde shuttles. They make sure to note that their operation is supported by one of the High Houses."

Reviewer 3: You probably don't _need_ to add this, but which High House is supporting their operation?

Author: The trouble is that specifying it narratively restricts where it could be offered, since it isn't likely that e.g. one of the High Houses is going to be advertising on the home planet of another.

Reviewer 3: That's a good point. Maybe leave it as is, but possibly add a couple briefer news items linked to the most prominent High Houses that only offer on their home planets?

	name
		word
			"Local News Report"
	message
		word
			"A broadcast quietly lists out the details of a recent weather control unit failure, which resulted in a brief evacuation but no injuries. A photo shows a few dozen Successors gathered together along a shoreline while several ships swarm in the distance."
			"A muted advertisement plays, showing architectural renders of several small underwater homes with walls that change color and texture from simulated wood, to simulated shell, to even a dark, amorphous metal resembling the hull of a starship."
			"A short broadcast discusses a recent thunderstorm, showing several colorful pictures of lightning and rain taken by locals during the event."
			"A text-only broadcast discusses a recent touch-based art performance. The script shimmers as you move your head around, the translation device doing its best to transcribe it."
			"A broadcast shows a brief interview with a local florist, who shows off a collection of their living-metal plants."
			"A formal document is shown with a column of text to the side of it. This describes it as a new trade ordinance, and lists its effects on independent merchants."

> 			"A brief program on erosion control and restoration plays for a duration."

Reviewer 3: You've already told me it's brief, I don't also need to know that it played for some ambiguous amount of time.
```suggestion
			"A brief program on erosion control and restoration plays."
```
You could also add another sentence, of course, if you felt like it.

> 			"The program is briefly interrupted to display a general-use map for the spaceport."

Reviewer 3: This one only makes sense in the context of the previous one.
```suggestion
			"The news report playing on the video screen is briefly interrupted to display a general-use map for the spaceport."
```

news "station report"
	to show
		has "language: Successor"
	location
		attributes successor
		attributes station
		not attributes uninhabited
	name
		word
			"Station Report"
			"Announcement System"

> 			"Public Announcement System"

Reviewer 3: Maybe:
```suggestion
			"Public Address System"
```

Reviewer 3: Maybe, but a public address system is a legitimate thing, while a google search for "public announcement system" redirects to public address systems.
